io/io/inc/LinkDef.h

```diff
@@ -50,7 +50,6 @@
 #pragma link C++ class TStreamerInfoActions::TConfiguredAction+;
 #pragma link C++ class TStreamerInfoActions::TActionSequence+;
 #pragma link C++ class TStreamerInfoActions::TConfiguration-;
-#pragma link C++ class ROOT::Internal::RRawFile+;
```

Comment:
@pcanal  do we need to add Lindkef file with ROOT::Internal::RRawFile class to ROOTIOLite?

Reply:
Update, I did it before but didn't notice this in all these lines of changes.